jest.config.js

```diff
@@ -1,9 +1,32 @@
-module.exports = {
+// @ts-check
+/** @type {import('@jest/types').Config.InitialProjectOptions} */
+const comonOptions = {
```

Comment:
Probably "commonOptions" ...
```suggestions
const commonOptions = {
```